tests/unit/array/invoke-test.js

```diff
@@ -0,0 +1,41 @@
+import { invoke } from 'ember-awesome-macros/array';
+import { module, test } from 'qunit';
+import compute from 'ember-macro-test-helpers/compute';
+
+let array;
+
+module('Unit | Macro | array | length', {
```

Comment:
Good tests, but I would move to the Integration section as this is testing real implementation, vs stubbing the function and asserting it was called like the other unit tests.